### DISCUSSION TITLE
Can't find the way to get a new token

### DISCUSSION THREAD

**Discussion by lopbox** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#656] [Q&A]
As it seems the token i first made expired. I don't know how to search for a new token, and as i search the web on how to get new Google api token i find tons of stuff to read but nowhere how to envoke the procedure to make one. Please help

**Comment by sumeshir26** [comment]
Click the button in the quick action bar

**Comment by Melvin-Abraham** [comment]
As mentioned by @sumeshir26, you may click on the "Reset Tokens" button on the suggestion/quick actions bar. You may be prompted to confirm the action. Once you confirm that, the assistant will reload and redirect you to the authorization code screen after which you may follow the instructions from 6th step in [**this** section of this wiki](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client#setting-authentication-in-the-app).

![Reset Tokens](https://user-images.githubusercontent.com/26056183/153993451-306fc01c-6959-40cf-bc26-a387d907b430.png)

**Reply by lopbox** [reply]
![Στιγμιότυπο από 2022-02-15 23-09-32](https://user-images.githubusercontent.com/1247044/154149645-aa1618c6-befe-404e-a59d-a3bdcad378d4.png)

i get this error without the prompt of reset tokens.

**Reply by lopbox** [reply]
i found the solution to my problem, i had to delete the existing token.json file and restart the app so that it would prompt me to get a token.
